src/components/TaskGroup/TaskGroup.module.scss

```diff
@@ -92,3 +100,9 @@
   max-height: 170px;
   overflow-y: auto;
 }
+
+@media screen and (max-width: $sm) {
```

Comment:
Did you try mixins ?
Asuming that we write css from mobile to desktop we could have example mixins(could be more - but from experience - do not overcomplicate that) - i.e 
```
@mixin tablet {
    @media (min-width: #{$tablet})) {
      @content;
  }
}

@mixin desktop {
    @media (min-width: #{$desktop})) {
      @content;
  }
}
```

then 
```
@include desktop {
 ...css 
}
```